assets/js/components/module/ModuleFooter.js

```diff
@@ -25,17 +25,22 @@ import classNames from 'classnames';
  * Internal dependencies
  */
 import HelpLink from '../HelpLink';
+import { useFeature } from '../../hooks/useFeature';
 
-const ModuleFooter = () => (
-	<div className="mdc-layout-grid__inner">
-		<div className={ classNames( [
-			'mdc-layout-grid__cell',
-			'mdc-layout-grid__cell--span-12',
-			'mdc-layout-grid__cell--align-right',
-		] ) }>
-			<HelpLink />
+const ModuleFooter = () => {
+	const helpVisibilityEnabled = useFeature( 'helpVisibility' );
+
+	return (
+		<div className="mdc-layout-grid__inner">
+			<div className={ classNames( [
+				'mdc-layout-grid__cell',
+				'mdc-layout-grid__cell--span-12',
+				'mdc-layout-grid__cell--align-right',
+			] ) }>
+				{ ! helpVisibilityEnabled && <HelpLink /> }
+			</div>
 		</div>
-	</div>
-);
+	);
+};
```

Comment:
For cases like here where `HelpLink` is the only element within the container / grid, the flag should be around the entire container / grid, to avoid ending up with unnecessary empty containers.

There might in theory be cases where some of the containers are needed to not break the layout, and if there is one like that, it's okay to keep there (we shouldn't change any CSS as part of this PR). However I'm pretty sure all of these containers that have _only_ a `HelpLink` are full-width (span 12) containers, so we should just return `null` from the entire component if the flag is disabled, rather than rendering an empty container.

**Update:** Due to another observation I made here (see other comment on this file), the change right here isn't needed. Still keeping this comment for context.

Reply:
That's true, I just figured it was just easier to grep for the conditional rendering by keeping it all the same (eg. `{ ! helpVisibilityEnabled && <HelpLink /> }`). Either way it renders more/less the same visually, but I've removed the containers now 🙂